command/node_drain.go

```diff
@@ -68,8 +68,15 @@ func (c *NodeDrainCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Check if node exists
+	node, _, err := client.Nodes().Info(nodeID, nil)
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error toggling drain mode: %s", err))
```

Comment:
I think this is fine as the `err` will be something like "node with id "foobar" not found"